### PR TITLE
BWDB-13907: Adding Porting tag to bulkPortins and tollFreePortingValidations endpoints

### PR DESCRIPTION
**Pull request by evanhalley**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

## Changes

I added the `Porting` tag to a handful of `bulkPortins` and `tollFreePortingValidations` endpoints.

## References
[BWDB-13907 "API: Treat Reserved, Assigned, and Suspended as portable states for phase 1 toll free port-ins"](https://bandwidth-jira.atlassian.net/browse/BWDB-13907)

### CODE REVIEW

File: site/specs-temp/numbers.json [hunk 1]
```diff
@@ -1701,7 +1701,8 @@
     "/accounts/{accountId}/bulkPortins": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:1705]:
```suggestion
          "Porting"
```
I think we want to get rid of `/accounts` in the tags 

unfortunately the docs renderer keys off of the tag and will duplicate this endpoint in the docs

File: site/specs-temp/numbers.json [hunk 2]
```diff
@@ -1818,7 +1819,8 @@
       },
       "post": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:1823]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 3]
```diff
@@ -1879,7 +1881,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:1885]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 4]
```diff
@@ -1922,7 +1925,8 @@
       },
       "put": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:1929]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 5]
```diff
@@ -1990,7 +1994,8 @@
       },
       "delete": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:1998]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 6]
```diff
@@ -2033,7 +2038,8 @@
       },
       "patch": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2042]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 7]
```diff
@@ -2103,7 +2109,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}/portinList": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2113]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 8]
```diff
@@ -2143,7 +2150,8 @@
       },
       "put": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2154]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 9]
```diff
@@ -2203,7 +2211,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}/notes": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2215]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 10]
```diff
@@ -2306,7 +2315,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}/notes/{noteId}": {
       "put": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2319]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 11]
```diff
@@ -2376,7 +2386,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}/tnList": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2390]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 12]
```diff
@@ -2489,7 +2501,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}/history": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2505]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 13]
```diff
@@ -2547,7 +2560,8 @@
     "/accounts/{accountId}/bulkPortins/{orderid}/loas": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2564]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 14]
```diff
@@ -2598,7 +2612,8 @@
       },
       "post": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:2616]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 15]
```diff
@@ -17149,7 +17164,8 @@
...
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:17168]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 16]
```diff
@@ -17271,7 +17287,8 @@
     "/accounts/{accountId}/tollFreePortingValidations/{orderId}": {
       "get": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:17291]:
```suggestion
          "Porting"
```

File: site/specs-temp/numbers.json [hunk 17]
```diff
@@ -17332,7 +17349,8 @@
       },
       "patch": {
         "tags": [
-          "/accounts"
+          "/accounts",
+          "Porting"
```
Comment (ajrice6713) [inline site/specs-temp/numbers.json:17353]:
```suggestion
          "Porting"
```

**Review by ajrice6713 — CHANGES_REQUESTED** [review]

**Review by ajrice6713 — APPROVED** [review]

**Comment by evanhalley** [comment]
Thanks for the review AJ, it was decided that we remove the Porting tag instead.

**Pull request closed without merging**